src/utils/persistentStore.ts

```diff
@@ -181,5 +176,26 @@ export const persistLogVisible = (visible: boolean) =>
         `isLogVisible`,
         visible
     );
+
+export const persistIsLoggingVerbose = (value: boolean) =>
+    getAppSpecificStore<SharedAppSpecificStoreSchema>().set(
+        'isLoggingVerbose',
+        value
+    );
+export const getIsLoggingVerbose = () =>
+    getAppSpecificStore<SharedAppSpecificStoreSchema>().get(
+        'isLoggingVerbose',
+        false
+    );
+
+const resetIsLoggingVerbose = () =>
+    getAppSpecificStore<SharedAppSpecificStoreSchema>().set(
+        'isLoggingVerbose',
+        false
+    );
```

Comment:
S: Reuse `persistIsLoggingVerbose` from above:
 ```suggestion
const resetIsLoggingVerbose = () => persistIsLoggingVerbose(false);
```